### PR TITLE
Travis and package optimizations

### PR DESCRIPTION
**Pull request by NejcZdovc**
- added travis parallel jobs
- remove duplicated node definitions
- replaces nsp with audit
- removed cpx as I couldn't fix it to be used
- renamed some tasks so that they are grouped together
- removed check security for push as it's expensive
- added trusted distribution
- addressed audit problems that were found
- updated package-lock
- removed babel as I don't think it's used

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,19 +12,28 @@
   ],
   "license": "MPL-2.0",
   "scripts": {
+    "ts": "tsc",
+    "ts-watch": "tsc -w",
+    "lint": "standard --verbose",
+    "lint-fix": "standard --fix --verbose",
     "bootstrap": "lerna bootstrap",
```
Comment (NejcZdovc) [inline package.json]: would love to group this 3 together as well. When do we use bootstrap and build?

Comment (NejcZdovc) [reply package.json]: @cezaraugusto

Comment (cezaraugusto) [reply package.json]: we're not using bootstrap anymore so it's more like a leftover, I'm ok grouping other tasks

File: docs/contributing.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ There are several ways you can do so:
 2. cd `brave-ui`
 3. `npm i` or `yarn install`
 
-Once the installation is finished, run `npm start`. The source is available under http://localhost:9091 (one after nine-o-nine?). It will hot-reload the style guide at every code change.
+Once the installation is finished, run `npm story-start`. The source is available under http://localhost:9091 (one after nine-o-nine?). It will hot-reload the style guide at every code change.
```
Comment (cezaraugusto) [inline docs/contributing.md]: per script it's `storybook-start` now, right?

File: package.json [hunk 3]
```diff
@@ -12,19 +12,28 @@
...
-    "publish": "lerna publish && npm run deploy",
-    "start": "start-storybook -p 9091 -c .storybook",
-    "test": "jest -t",
-    "test-all": "jest --verbose",
+    "publish": "lerna publish && npm run storybook-deploy",
+    "storybook-start": "start-storybook -p 9091 -c .storybook",
+    "storybook-deploy": "storybook-to-ghpages -s public -o .out",
+    "test-unit": "jest -t",
+    "test-unit-all": "jest --verbose",
+    "test-security": "npm audit",
```
Comment (cezaraugusto) [inline package.json]: cool stuff

File: docs/contributing.md (hunk 2 above)
Comment (NejcZdovc) [reply docs/contributing.md]: fixed

File: package.json (hunk 1 above)
Comment (NejcZdovc) [reply package.json]: removed

**Pull request merged**